api/app/src/test/kotlin/packit/integration/controllers/LoginControllerTest.kt

```diff
@@ -18,9 +22,11 @@ class LoginControllerTest : IntegrationTest()
     }
 
     @Test
-    fun `can login with API`()
+    fun `can login with github API`()
```

Comment:
Could add an integration test for hitting the basic endpoint too

Reply:
i couldnt unless im being stupid.. but requires a change in config.properties file.. which is actually not even a spring loaded one.... so I had an idea of moving config properties all into the spring application.properties so they can be controlled in tests

Reply:
Oh yeah, that's a pain. Let's make a separate ticket to do that so we can have a fuller set of integration tests.